### ISSUE TITLE
expand_paths for `__main__` runner

### ISSUE THREAD

**Issue by gerrymanoim** [isidentical/refactor#21]
It looks like the runner in main.py passes `options.src` to `run_files` directly https://github.com/isidentical/refactor/blob/master/refactor/__main__.py#L62 so you must pass in a single python file whereas the unbound runner expands the paths so you  can run this on a directory of files: https://github.com/isidentical/refactor/blob/master/refactor/runner.py#L89-L91. 

Would you accept a PR to change this behavior? Or was this separation purposeful?

**Comment by isidentical** [comment]
The initial assumption was that, you would pass a single file which would load all the refactoring rules, e.g:
```
# .refactors/__init__.py
from .my_rule import MyRule
from .my_rule2 import MyRule2
```

But if there is a use case, I think it would be acceptable. The only thing that should be carefully handled is that, when a user provides multiple files (`python -m refactor a.py b.py`) we should not load the duplicate rules. So after the results are mapped to `get_refactors`, we should probably cast the rules into a set first.

**Comment by gerrymanoim** [comment]
Sorry - I think I'm missing something. What you said above makes sense to me, but I was talking about the source directory of files you want to refactor.

**Comment by isidentical** [comment]
> Sorry - I think I'm missing something. What you said above makes sense to me, but I was talking about the source directory of files you want to refactor.

Ah, that makes perfect sense. Let's do it!

**Comment by isidentical** [comment]
Fixed by #22
